### ISSUE TITLE
Unwanted extra click in resource cost history

### ISSUE THREAD

**Issue by jeffnm** [Coral-erm/Coral#201]
From a user:

In the Acquisitions tab, when editing Cost History, you have to click Add after entering data, otherwise it won't save. Since I never have the need to add a second line, which is what the Add button does, can that extra click be removed?

**Comment by rtatterson** [comment]
I agree; I can't think of reasons the extra click is useful so might want to consider this for removal.

**Comment by rtatterson** [comment]
Isn't present in 2025.04.
